webapp/components/DataQuery/Visualizer/Visualizer.js

```diff
@@ -1,37 +1,26 @@
-import './visualizer.scss'
 import React from 'react'
 import PropTypes from 'prop-types'
+
 import { Query } from '@common/model/query'
 
-import ProgressBar from '@webapp/components/progressBar'
 import Table from './Table'
 
 const components = {
   [Query.displayTypes.table]: Table,
 }
 
 const Visualizer = (props) => {
-  const { query, data, dataEmpty, dataLoaded, dataLoading, nodeDefsSelectorVisible, offset } = props
-
-  if (!dataLoading && !dataLoaded) return null
+  const { query, data, dataEmpty, nodeDefsSelectorVisible, offset } = props
 
   return (
     <div className="table__content">
-      {dataLoading && !dataLoaded ? (
-        <ProgressBar
-          className="running progress-bar-striped visualizer__progress-bar"
-          progress={100}
-          showText={false}
-        />
-      ) : (
-        React.createElement(components[Query.getDisplayType(query)], {
-          query,
-          data,
-          dataEmpty,
-          nodeDefsSelectorVisible,
-          offset,
-        })
-      )}
+      {React.createElement(components[Query.getDisplayType(query)], {
+        query,
+        data,
+        dataEmpty,
+        nodeDefsSelectorVisible,
+        offset,
+      })}
```

Comment:
are these values going to be the same in each kind of component?

Reply:
at the moment we have only one (Table) and that's what it needs.
But I guess yes, when we'll add other visualization components they won't need more than that